pumpkin-lib/src/propagators/cumulative/time_table/time_table_over_interval.rs

```diff
@@ -51,6 +51,7 @@ pub(crate) struct Event<Var> {
 /// \[1\] A. Schutt, Improving scheduling by learning. University of Melbourne, Department of
 /// Computer Science and Software Engineering, 2011.
 #[derive(Debug)]
+#[allow(unused)]
```

Comment:
Why? If it is unused we should probably not include it. Otherwise, is it a flag? And why was this not here before?

Reply:
This will be fixed by another PR, at the moment we don't have support for selecting the type of propagator used for propagating the cumulative constraint!